README.md

```diff
@@ -157,6 +157,8 @@ A `report` is a JSON structure with the following entries:
 | `title`        | REQUIRED | A one-line title describing this report.| 
 | `score`        | REQUIRED | The severity of this report from -100 to 100, with 100 most critical.| 
 | `iocs`         | REQUIRED | The IOCs for this report.  A match on __any__ IOC will cause the activity to be tagged with this report id.  The IOC format is described below.| 
+| `tags`         | OPTIONAL | A comma separated list of identifiers to tag the report. |
+| `description`  | OPTIONAL | A description of the report. |
```

Comment:
Perfect. I'm going to update the docs on our Developer Network website as well: http://developer.carbonblack.com/reference/enterprise-response/5.1/threat-intelligence-feeds/

BTW, this README will be replaced with a link to that page once the website is "officially" announced